datapackage_pipelines_aws/processors/to_s3.py

```diff
@@ -0,0 +1,28 @@
+import botocore
+import boto3
+import os
+
+from datapackage_pipelines.lib.dump.dumper_base import CSVDumper
+from datapackage_pipelines_aws import helpers
+
+class S3Dumper(CSVDumper):
+
+    def initialize(self, params):
+        super(S3Dumper, self).initialize(params)
+        self.bucket = params['bucket']
```

Comment:
@akariv boto3 takes care of credential if they are set-up. It looks up in aws config file or searches for ENV variables http://boto3.readthedocs.io/en/latest/guide/configuration.html#aws-config-file
But we can have them as a part of the spec as well if that is a case